src/java/io/bazel/rulesscala/scalac/ReportableMainClass.java

```diff
@@ -11,15 +11,28 @@
 import scala.tools.nsc.MainClass;
 import scala.tools.nsc.Settings;
 import scala.tools.nsc.reporters.Reporter;
+import java.lang.AutoCloseable;
 
 public class ReportableMainClass extends MainClass {
   private Reporter reporter;
   private final CompileOptions ops;
+  private Global _compiler = null;
 
   public ReportableMainClass(CompileOptions ops) {
     this.ops = ops;
   }
 
+  public void Close() throws Exception{
```

Comment:
style nit: don't start method names with a capital letter. Please rename to `close()`

Reply:
done